types/era_info.go

```diff
@@ -19,7 +19,7 @@ type EraInfo struct {
 // SeigniorageAllocation sores information about a seigniorage allocation
 type SeigniorageAllocation struct {
 	Validator *ValidatorAllocation `json:"Validator,omitempty"`
-	Delegator *DelegatorAllocation `json:"Delegator,omitempty"`
+	Delegator *DelegatorAllocation `json:"DelegatorKind,omitempty"`
```

Comment:
Shouldn't we have both `Delegator` and `DelegatorKind` keys to support both 1.5.x and 2.0?